internal/sequencer/helpers.go

```diff
@@ -133,3 +145,50 @@ func privateKeyFromText(privkey string) (ed25519.PrivateKey, error) {
 	from := ed25519.NewKeyFromSeed(privKeyBytes)
 	return from, nil
 }
+
+// getAddressAsBytes converts an address string to a byte array. It will first
+// attempt to decode the address as bech32m, and if that fails, it will attempt
```

Comment:
Can we instead check if the address starts with `BechAddressPrefix` to determine if it's bech?

Could have another func `isBech` .

Reply:
Is there actually a non zero chance that an ed25519 address could start with the string "astria"? Maybe my suggestion won't work.

Reply:
There's no chance an ed25519 address could start with "astria", but if for some odd reason we ever change our name to a string of valid hex characters, it could be an issue? Haha